README.md

```diff
@@ -16,15 +17,138 @@ TerseTS can be compiled and cross-compiled from source:
    - macOS: `zig build -Dtarget=aarch64-macos -Doptimize=ReleaseFast`
    - Microsoft Windows: `zig build -Dtarget=x86_64-windows -Doptimize=ReleaseFast`
 
-# Usage
-TODO
+## Usage
+TerseTS provides a C-API that is designed to be simple to wrap. Currently, TerseTS includes APIs for the following programming languages which can be used without installation of any dependencies:
+<a id="zig-usage-example"></a>
+<details>
+<summary><strong>Zig Usage Example</strong></summary>
 
-# Bindings
-TerseTS provides a C-API that is designed to be simple to wrap. Currently, TerseTS includes bindings for the following programming languages which can be used without installation of any dependencies:
-- [Zig](src/tersets.zig)
-- [C](bindings/c/tersets.h)
-- [C++](bindings/c/tersets.h)
-- [Python](bindings/python/tersets) using [ctypes](https://docs.python.org/3/library/ctypes.html)
+```c
```

Comment:
Maybe change to Zig for more correct syntax highlighting [if it is supported](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#syntax-highlighting).

Reply:
I checked before, and they do not support Zig. C was providing quite nice syntax highlighting compared to other languages that I tried before.